backend/pyproject.toml

```diff
@@ -82,6 +81,7 @@ pypdf2 = "^1.26.0"
 refextract = "^1.0.3"
 fs = "0.5.4"
 invenio-oaiserver = {git = "https://github.com/inspirehep/invenio-oaiserver.git", rev = "5d796cf354467773a7e00b75bc382ba2bf2efef7"}
+invenio-records-rest = {git = "https://github.com/inspirehep/invenio-records-rest.git", rev = "f398bab4f4f96607045e753d25aa09bb404e9e42"}
```

Comment:
just for reference 

```shell
poetry add git+https://github.com/inspirehep/invenio-records-rest.git#f398bab4f4f96607045e753d25aa09bb404e9e42
```

According to the documentation [documentation](https://python-poetry.org/docs/cli/#add), you have to explicitly specify the `SHA` with `#SHA`